scripts/fetch/fetch.precip_cells.R

```diff
@@ -0,0 +1,16 @@
+fetch.precip_cells <- function(viz){
+  library(sf)
+  library(sp)
```

Comment:
It'd be helpful to see sf:: and sp:: prepended on function calls instead of library() calls. I recognize that sometimes library() is needed too, but the prepended form is really nice to follow what depends on what in line.